lms/djangoapps/ytp/management/commands/update_user_fullname.py

```diff
@@ -91,7 +91,7 @@ def handle(self, *args, **options):
                                            )
                                        )
                            except Exception as e:
-                               if hasattr(e, 'message'):
+                               if hasattr(e, 'message') and e.message:
                                    error_string = e.message
                                else:
                                    error_string = str(e)
```

Comment:
Could you add traceback information to the log output?  

i.e. 
sys.exc_info()

Reply:
I added traceback information. Thank you for advice.